src/sushi-hooks/queries/useTokenBalance.ts

```diff
@@ -17,7 +17,7 @@ export interface BalanceProps {
 const useTokenBalance = (tokenAddress?: string, accountAddress?: string) => {
   const [balance, setBalance] = useState<BalanceProps>({ value: BigNumber.from(0), decimals: 18 })
   const { account } = useActiveWeb3React()
-  const [targetAddress] = useState(accountAddress ?? account)
+  const [targetAddress, setTargetAddress] = useState(account)
```

Comment:
@bill-ov I only mentioned this line to use as basis for my comment on line 40. Let's keep the original logic here pls

```suggestion
  const [targetAddress, setTargetAddress] = useState(accountAddress ?? account)
```

Reply:
Okay James. I'll change it after lunch. Sorry for not noticing your comment early

Reply:
Done changing it James

Reply:
Thanks @bill-ov ! However the linters & unit tests seems to be failing now. Can you pls take care of this? No need to do it now, but will leave it up to you. Not a priority

Reply:
Hi @schystz. Done resolving the issue. All tests are now passed